templates/python/custom-copilot-basic/src/config.py.tpl

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import os
+
```

Comment:
The logic of loading `.env` file is hidden in the `debugpy` task, it may be better to expose it to users. There're 2 options:
1. Update `debugpy` task in `launch.json` to specify the `.env` file explicitly
2. Leverage `python-dotenv` package to load the `.env` file in `config.py` explicitly.

Both option works. You could choose one. I personally prefer `2` because it also works when user runs the app outside of VS Code.